### PR TITLE
Add some shadow to our widgets

### PR DESCRIPTION
**Pull request by sauldom102**
## 🔑 What?

Add some shadow to our widgets.

## 👁️ Visual proof

Screenshot 2024-09-27 at 17 17 25

## 🏡 Context

- [💼 Jira](https://factorialmakers.atlassian.net/browse/FCT-19213)

### CODE REVIEW

File: lib/experimental/Widgets/Layout/Dashboard/index.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ const DashboardComponent = forwardRef<HTMLDivElement, DashboardProps>(
     }, [setColumns])
 
     return (
-      <div ref={ref} className="overflow-hidden">
```
Comment (sauldom102) [inline lib/experimental/Widgets/Layout/Dashboard/index.tsx:42]: This is needed to prevent shadows to be cropped on the sides

**Review by nlopin — APPROVED** [review]

**Comment by josepjaume** [comment]
This looks cool but unfortunately it breaks the WidgetStrip a bit 😢 . 

image

The whole "Widgets can be scrolled horizontally" looks like isn't going anywhere (haven't seen it on the latest designs) so maybe we can ditch it in favor of simply a responsive behaviour.

File: lib/experimental/Widgets/Layout/Dashboard/index.tsx (hunk 1 above)
Comment (josepjaume) [reply lib/experimental/Widgets/Layout/Dashboard/index.tsx:42]: Unfortunately this also means that we are dropping the scroll bars 👀

Comment (nlopin) [reply lib/experimental/Widgets/Layout/Dashboard/index.tsx:42]: can we add some padding to accommodate the shadows?

Comment (josepjaume) [reply lib/experimental/Widgets/Layout/Dashboard/index.tsx:42]: But then we would need to add negative margin to the container. I think the best solution, as spoken with @sauldom102 is to make the parent (container) dictate the shadows via `[&>*]:shadow-md`. The dashboard benefits from shadows, but a strip doesn't.

If you think about it, it makes sense that this is the responsability of the parent, the same way gap or overall layout is.

Comment (josepjaume) [reply lib/experimental/Widgets/Layout/Dashboard/index.tsx:42]: Sorry, I'm mixing things:

1. Yes we need to remove `overflow-hidden` from dashboard, 100%
2. We can't have widgets have shadows, we need to let the container decide.

**Pull request merged**